### PR TITLE
Fix most lint warnings

### PR DESCRIPTION
**Pull request by Ef55**

### CODE REVIEW

**Review by justagoodguy-omega — APPROVED** [review]
Thank you for your sacrifice, your country is proud

File: app/build.gradle [hunk 1]
```diff
@@ -52,18 +52,18 @@ def getApiKey() {
...
     implementation 'androidx.navigation:navigation-fragment:2.3.5'
     implementation 'androidx.navigation:navigation-ui:2.3.5'
     implementation 'androidx.lifecycle:lifecycle-livedata-ktx:2.3.1'
     implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.3.1'
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
-    implementation 'androidx.recyclerview:recyclerview:1.2.0'
+    implementation 'androidx.recyclerview:recyclerview:1.2.1'
     implementation 'androidx.cardview:cardview:1.0.0'
-    implementation 'com.google.android.material:material:1.4.0-alpha02'
+    implementation 'com.google.android.material:material:1.4.0-beta01'
```
Comment (TheImpressiveDonut) [inline app/build.gradle:66]: hmm git hub is telling there is a lint warning here

**Review by TheImpressiveDonut — CHANGES_REQUESTED** [review]
Nice clean up ! There still some easy fixable lint warnings like hard coded string ! I pointed out on of them which a library warning.

**Review by TheImpressiveDonut — APPROVED** [review]
So fix my concerns myself ! LOL

**Pull request merged**